homeassistant/components/camera/proxy.py

```diff
@@ -162,8 +233,12 @@ def camera_image(self):
             _LOGGER.error("Error getting original camera image")
             return self._last_image
 
+        if self._mode == MODE_RESIZE:
+            job = _resize_image
+        else:
+            job = _crop_image
         image = await self.hass.async_add_job(
-            _resize_image, image.content, self._image_opts)
+            job, image.content, self._image_opts)
```

Comment:
`async_add_executor_job`

Reply:
Thanks for reviewing this. Could you please explain the difference between the two?
(To be noted that the original code already had `async_add_job`)

Reply:
that is the new style

Reply:
Thanks. The question is relevant also for another platform I'm coding, so I'll try and see what has changed in those methods.